--- 
authors: Freek Pols
updated: december 3, 2025    
---
# Bepaling van soortelijke warmte van water


# Introductie

Water heeft een enorme buffercapaciteit: je moet veel energie toevoegen om de temperatuur van water een graad te verwarmen. In dit practicum gaan we de soortelijke warmte van water bepalen door een bekende hoeveelheid water te verwarmen met een bekende hoeveelheid energie, en de temperatuurstijging te meten.

# Theorie

De soortelijke warmte van water is de hoeveelheid warmte die nodig is om een kilogram aan water 1 graad Celsius of Kelvin te verhogen en is te berekenen met $$c =\frac{Q}{m\Delta T} $$. In het practicum wordt er energie toegevoegd met behulp van en voedingsbron. De energie die toegevoegd wordt staat gelijk aan $$ Q = P \cdot t$$.
Waarbij $$ P = U \cdot I$$.


# Methode en materialen

## Ontwerp
Een waterbad met bekende massa aan water wordt verwarmd met een elektrisch verwarmingselement dat een bekende hoeveelheid energie levert.
De temperatuur van het water wordt gemeten met een temperatuursensor.
Door de temperatuurstijging als functie van de tijd te meten kan de soortelijke warmte van water worden berekend.


## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Maatbeker
- Weegschaal
- Water
- Elektrisch verwarmingselement ($10 \mathrm{\Omega}$, $10 \mathrm{W}$)
- Voedingsbron 
- Thermometer of temperatuursensor
- Stopwatch of timer

```{figure} figures/cwater_setup.jpg
:width: 70%

Een schematische weergave van de opstelling
```

## Procedure
```{exercise}
1. Beschrijf de procedure die nodig is om de soortelijke warmte van water te bepalen.
2. Geef jouw beschrijving van de procedure aan een ander team, zij verzamelen voor jou de data op basis van de gegevens.
3. Voer het experiment van een ander team uit op basis van hun beschrijving.
4. Analyseer de data die het andere groepje voor jou heeft verzameld.
```


## Veiligheid
We maken gebruik van een $10 \mathrm{\Omega}$, $10 \mathrm{W}$ weerstand.
Deze wordt snel heet.
De bronspanning mag dan ook alleen aan wanneer de weerstand in het water zit.
Raak de weerstand niet aan tijdens het experiment.
Omdat de weerstand in het water zit, kunnen we wel het elektrisch vermogen hoger zetten zonder dat de weerstand oververhit raakt.
Het maximaal vermogen mag $40 \mathrm{W}$ zijn.
Daarbij moet de roerder wel aanstaan om de warmte goed te verdelen.

## Data analyse
Geef kort de data-analysemethode weer.


De procedure :Weeg eerst de maatbeker en vul hem tot 600 mL en weeg daarna de maatbeker met water.
- Meet het begin temperatuur van het water en leg de elektrisch verwarmingselement in het water.
- Verbind de rode draad van het elektrischverwarmingselement aan de plus gat van de voedingsbrong en de zwarte bij de min kant.
- Zet de stroom op 2 A en doe 2 verschillende metingen met 7.5V en 15V
- Verbind de voedingsbron parralel met de multimeter om de spanning te meten
- Start de timer tegelijk met de verbinding met de voedingsbron en noteer per graad omhoog de tijdstip tot het 5 graden omhoog is gegaan.
- Bepaal met behulp van P= U*I en E=P*t de energie
- Bereken de c met de Energie waardes met behulp c = Q/m delta T  

De data-analyse methode:
Door de massa van de maatbekere met water af te trekken van de massa van de maatbeker krijg je de massa van het water. Door de gemeten tijd keer het vermogen wat je krijgt door de voltagen keer de stroom te doen verkijg je de energie. De energie bereken je voor alle 5 de tijdstippen en vul je dan in de formule om de soortelijke warmte te berekenen waarbij dus het temperatuur verschil dus 1,2,3,4 en 5 is. Het gemiddelde van deze waardes is het resultaat.

# Resultaten



In [1]:
# Hier de data en de analyse
# Metingen waren alleen gedaan met 15 V en 1.5 A vanwege wat wel en niet mocht van de T.A. voor de veiligheid en behoud van de apparatuur
import numpy as np
U = 15
I= 1.5
Massa_beker = 0.2824
Massa_beker_water = 0.8807
massa_water = Massa_beker_water - Massa_beker
Begintemperatuur = 19.5 
t = np.array([101, 219, 329, 440, 588])
P = U*I
Q = P*t
delta_T = np.array([1,2,3,4,5])
c_ =  np.divide(Q,massa_water*delta_T)
c= np.average(c_)
print('De soortelijke warmte van water is ',round(c),'J/kgK')


De soortelijke warmte van water is  4120 J/kgK


In [2]:
# Sla figuren op met  
# 
# plt.savefig("figures/naam.png", dpi=450)
# Met de methode die  er gerbuikt is was er geen plot gemaakt dus is er geen figuur om op te slaan.



# Discussie en conclusie

Hier een korte discussie en conclusie over de resultaten van het experiment en de implicaties daarvan.


Uit dit practicum is de gevonden waarde voor de soortelijke warmte van water 4120 J/kgK. Dit ligt dicht bij de theoretische waarde van 4186 J/kgK, dus het experiment was redelijk succesvol.
Het tijdverschil tussen opeenvolgende temperatuurstijgingen hoort volgens de theorie constant te zijn (Δt zou gelijk moeten zijn voor iedere graad stijging), maar uit de resultaten blijkt dat dit niet het geval is. De tijd neemt toe naarmate de temperatuur stijgt.
Een waarschijnlijke oorzaak hiervan is dat het warmteverlies naar de omgeving groter wordt naarmate het temperatuurverschil tussen het water en de omgeving toeneemt. Dit volgt uit de wet van Newton voor afkoeling:
$$Q_{verlies} = h \cdot A \cdot \Delta T_{omgeving}$$

Omdat de temperatuur van het water stijgt, wordt ΔT groter, dus groter warmteverlies. Dit betekent dat niet alle elektrische energie in het water gaat, maar een deel verloren gaat aan de omgeving.Voor een verbeterd onderzoek plaats het waterbad in een geïsoleerde container en Herhaal het experiment meerdere keren.

